package.json

```diff
@@ -5,7 +5,7 @@
     "lint": "eslint --cache **/*.{js,mjs,jsx,ts,tsx}",
     "lint-fix": "npm run lint -- --fix",
     "prettier": "prettier --write **/*.json **/*.yml src/instruments/**/*.css",
-    "build:instruments": "rollup -c src/instruments/rollup.config.js",
+    "build:instruments": "cd src/instruments && ./build.sh",
     "serve:efb": "cd src/instruments/src/EFB/ && rollup -wc ./rollup.development.config.js",
     "build:fbw": "cd src/fbw && ./build.sh",
     "build:fadec": "cd src/fadec && ./build.sh"
```

Comment:
Why are these needed? This can be done by passing a regex to igniter instead...

Reply:
Forgot to remove them